core/src/env/srml/srml_only/impls.rs

```diff
@@ -73,12 +72,33 @@ impl EnvStorage for SrmlEnvStorage {
     }
 }
 
+/// Wrapped access to return data.
+/// get() or get_raw Must be used right after ReturnData is returned.
+#[allow(unused)]
+pub struct ReturnData<T> {
+    marker: PhantomData<fn() -> T>,
+}
+
+#[allow(unused)]
+impl<T> ReturnData<T>
+where
+    T: Decode,
+{
+    fn get() -> T {
```

Comment:
Both `get` and `get_raw` should take `self` as their first argument to make them only callable once.
```suggestion
    fn get(self) -> T {
```